### ISSUE TITLE
Error running aggregate on Java SDK(JVM) variants.

### ISSUE THREAD

**Issue by akshay-ap** [opendatakit/aggregate#155]
#### Software and hardware versions 
Linux mint 18.1 (64 bit), Aggregate v1.2, IBM SDK, Java Technology Edition, Version 8.0

#### Problem description
I was trying to setup aggregate server on my local machine using IBM java 8 as Java SDK on IntellJ IDE. The daemon process was unable start.

Message shown :
`Executing external task 'appStartWar'...`
`Gradle build daemon disappeared unexpectedly (it may have been killed or may have crashed)`
`External task execution finished 'appStartWar'.`
Here is screenshot in first attempt : 
![agg3](https://user-images.githubusercontent.com/16358982/33231614-28647fe2-d21e-11e7-8d63-b5b57b78d67d.png)

I removed IBM jvm and installed it again.
In second attempt build started but couldn't proceed. Here is the screen shot in second attempt.
![agger](https://user-images.githubusercontent.com/16358982/33231595-bf902af2-d21d-11e7-81b9-362599e5f618.png)

#### Steps to reproduce the problem
Install IBM jvm. Configure IntellJ IDE with IBM JVM.

`File -> Other settings -> Default Project Structure -> Platform Settings -> SDKs`

![aggr4](https://user-images.githubusercontent.com/16358982/33231643-e63a957e-d21e-11e7-9463-f0478671d254.png)

(Remove any existing Java SDK in IntellJ)

Trying running the task `appStartWar`

#### How did I fix this?
sudo update-alternatives --config java
sudo update-alternatives --remove "java" /usr/lib/jvm/java-ibm-x86_64-80/jre/bin/java
sudo rm -r /usr/lib/jvm/java-ibm-x86_64-80/
sudo apt-get install openjdk-8-jdk

Restart IntellJ IDE.
Changed the Java SDK (JVM/JDK) configuration.
![agg5](https://user-images.githubusercontent.com/16358982/33231747-5c585b00-d220-11e7-8a5a-37addc28e4ae.png)

#### Other information 
Things you tried, stack traces, related issues, suggestions on how to fix it...
In short it seems like running `aggregate` on other Java SDK variants might fail.
The `readme.md` file should contain information about recommended Java SDK versions.
A guide on configuring Java SDK (JVM/JDK) for IntellJ IDE will be helpful in `readme.md`.

Unfortunately, I didn't have source of IBM java SDK version for first attempt. (I had installed it long back.)

I downloaded IBM Java SDK for second attempt from here.
https://www-01.ibm.com/marketing/iwm/iwm/web/pick.do?source=swg-sdk8&S_PKG=amd64_8.0.3.12&S_TACT=105AGX05&S_CMP=JDK&lang=en_US

**Comment by ggalmazor** [comment]
So, for starters, I'd suggest adding some notice in the `README.md` file, stating that in order to avoid this kind of problems, you should use Oracle's or OpenJDK's JVM, which are officially supported.
